doc/docker/varnish.yml

```
@@ -9,7 +9,7 @@ services:
   app:
     environment:
      - SYMFONY_HTTP_CACHE=0
-     - SYMFONY_TRUSTED_PROXIES=varnish
+     - SYMFONY_TRUSTED_PROXIES=172.16.238.21 # Hardcoded IP of Varnish container
      - HTTPCACHE_PURGE_SERVER=http://varnish
      - HTTPCACHE_PURGE_TYPE=http
```

Nitpick to not use the deprecated name:
```suggestion
     - HTTPCACHE_PURGE_TYPE=varnish
```